app/src/main/res/layout/screen_wallet_connect_safe_selection.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```

Comment:
Why do we have this additional CoordinatorLayout? Is it not enough just to have ConstraintLayout as root here?

Reply:
this coordinator layout makes snackbars dismisable (via swipe)